src/pages/FundYield/FundYield.tsx

```diff
@@ -42,8 +42,7 @@ const FundYield = () => {
 
   return (
     <div className="Exchange page-layout">
-      <FundBanner />
-      <div className="Exchange-content">
+      {/* <div className="Exchange-content">
```

Comment:
Why are you commenting this out? Just remove the banner and it will show up fine

Reply:
The whole component was giving some errors, not else.